src/pages/account/logins/[type]/new.tsx

```diff
@@ -1,14 +1,62 @@
+import { HttpStatus } from "@fewlines/fwl-web";
+import {
+  JsonWebTokenError,
+  NotBeforeError,
+  TokenExpiredError,
+} from "jsonwebtoken";
 import { GetServerSideProps } from "next";
 import React from "react";
 
-const NewIdentity: React.FC = () => {
-  return <React.Fragment />;
+import Sentry, { addRequestScopeToSentry } from "../../../..//utils/sentry";
+import { IdentityTypes } from "../../../../@types/Identity";
+import { AddIdentity } from "../../../../components/business/AddIdentity";
+import { AddIdentityInputForm } from "../../../../components/display/fewlines/AddIdentityInputForm";
+import { withSSRLogger } from "../../../../middleware/withSSRLogger";
+import withSession from "../../../../middleware/withSession";
+
+const NewIdentity: React.FC<{ type: IdentityTypes }> = (props) => {
+  return (
+    <AddIdentity type={props.type}>
+      {({ addIdentity }) => (
+        <AddIdentityInputForm addIdentity={addIdentity} type={props.type} />
+      )}
+    </AddIdentity>
+  );
 };
 
 export default NewIdentity;
 
-export const getServerSideProps: GetServerSideProps = async (_context) => {
-  return {
-    props: {},
-  };
-};
+export const getServerSideProps: GetServerSideProps = withSSRLogger(
```

Comment:
Food for thought. When we'll do the middleware composition, we will need to abstract all the sentry/redirect part.